src/wiktextract/extractor/simple/page.py

```diff
@@ -0,0 +1,125 @@
+import re
+from typing import Any
+
+from wikitextprocessor.parser import LEVEL_KIND_FLAGS, print_tree
+from wiktextract.page import clean_node
+from wiktextract.wxr_context import WiktextractContext
+from wiktextract.wxr_logging import logger
+
+from .etymology import process_etym
+from .models import WordEntry
+from .pos import process_pos
+from .pronunciation import process_pron
+from .section_titles import POS_HEADINGS
+
+
+def parse_page(
+    wxr: WiktextractContext, page_title: str, page_text: str
+) -> list[dict[str, Any]]:
+    """Parse Simple Wiktionary page."""
+    # Unlike other wiktionaries, Simple Wikt. has a limited scope: only English
+    # words. The pages also tend to be much shorter and simpler in structure,
+    # which makes parsing them much simpler.
+    # https://simple.wiktionary.org/wiki/Wiktionary:Entry_layout_explained
+
+    if page_title in (
+        "Main Page",
+        "Main page",
+    ) or page_title.startswith("Appendix:"):
+        return []
```

Comment:
```suggestion
    if page_title == "Main Page" or page_title.startswith("Appendix:"):
        return []
```

redirect pages are filtered in `page_handler()`: https://github.com/tatuylonen/wiktextract/blob/a4f804451f4fe1216067c423bdc53a43ee7f38de/src/wiktextract/wiktionary.py#L55

`wxr.wtp.start_page()` is also called in that function.